auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -314,13 +315,23 @@ export default defineComponent({
       if (!documentKey || !documentName) return // safety check
 
       state.isDownloading = true
-      await BusinessService.downloadDocument(documentKey, documentName).catch(error => {
+      try {
+        if (DRS_ID_PATTERN.test(documentKey)) {
```

Comment:
do we need a feature flag for this, or can it be pushed out to prod next week if needed etc?

Reply:
@seeker25 
I changed the base branch since we are holding the updates on feature branch on `create-ui` repo.